posts/where_to_start_with_cloud_migration.md

```diff
@@ -0,0 +1,96 @@
+# Where to start with cloud migration?
+
+Your organisation has decided it's ready to move to the cloud. Where do you start?
+
+Building out a roadmap isn't as simple as writing a list of servers that need migrating. Cloud migrations are very rarely a one to one lift and shift from on-premise to a cloud provider like Amazon Web Services.
+
+Not only are migrations not as simple as lift and shift, the prioritisation of what to move and when is fundamental to making a migration successful. Knowing how to phase the migration into many small steps is crucial for reducing risk.
+
+There are a number of questions that can help inform your roadmapping process.
+
+## What are your mission critical applications?
+
+Changes to mission critical applications will have significant risks associated, 
+will likely impact other applications, and will potentially be the most disruptive to the organisation. It is for these reasons that mission critical applications should be tackled first.
+
+If your business sells products and services that are self serviced by customers then your e-commerce system will likely be mission critical. Less obvious self service systems used by customers such as a trading platforms or bespoke software, perhaps even running in your customers premises, are also likely to be critical.
```

Comment:
I'd reference something other than e-commerce here, as ecom is modern + 'web' era